frameworks/hdfs/universe/config.json

```diff
@@ -63,55 +63,59 @@
           "enum": ["OFF", "FATAL", "ERROR", "WARN", "INFO", "DEBUG", "TRACE", "ALL"],
           "default": "INFO"
         },
-        "tls": {
+        "security": {
+          "description": "HDFS security settings.",
           "type": "object",
-          "description": "TLS configuration properties.",
-          "properties": {
-            "enabled": {
-              "description": "Enable TLS support (requires Strict mode).",
-              "type": "boolean",
-              "default": false
+          "tls": {
+            "type": "object",
+            "description": "TLS configuration properties.",
+            "properties": {
+              "enabled": {
+                "description": "Enable TLS support (requires Strict mode).",
```

Comment:
It was my understanding that this is an EE feature, but does work on permissive.

Reply:
Yeah, possibly stale documentation, removed strict wording.